packages/cli/src/util/telemetry/commands/blob/put.ts

```diff
@@ -15,6 +15,10 @@ export class BlobPutTelemetryClient
     }
   }
 
+  trackCliInputSourceStdin() {
+    this.trackCliFlag('input-source-stdin');
```

Comment:
mhm, this isn't really a flag is it? Can we track this differently?

Reply:
To be honest I have 0 idea where this data ends up. I just did a search on trackCli* and it's not obvious which one to use.

I've updated it to:
```
  trackCliInputSourceStdin() {
    this.trackCliArgument({
      arg: 'pathToFile',
      value: '__vercel_stdin__',
    });
  }
```

Thoughts?